### ISSUE TITLE
Translator Aun Loakhe wrong emote

### ISSUE THREAD

**Issue by cyberkiller6** [ACEmulator/ACE#1456]
### Bug Report or Feature Request (mark with an `x`)
```
- [X] bug report -> please search issues before submitting
- [ ] feature request
```

### Repro steps.
Translator Aun Loakhe

https://imgur.com/a/4W6roJE

Your location is: 0xC6A9001B [94.144722 62.909084 71.173004] 0.636468 0.000000 0.000000 0.77130

### Desired functionality.
He should be banging his drum, not doing snowflake emote

### Mention any other details that might be useful.
https://asheron.fandom.com/wiki/Translator_Aun_Laokhe

**Comment by cyberkiller6** [comment]
Tested on commit #1454 and ACE World Database v0.9.18

**Comment by cyberkiller6** [comment]
Confirmed fixed in ACE World Database v0.9.19
